packages/odyssey-react-mui/src/@types/properties.d.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare module "properties" {
+  type ParseCallback = (error: Error, propertiesJson: object) => void;
+
+  type StringifyCallback = (error: Error, propertiesString: string) => void;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function createStringifier(): any;
```

Comment:
Not sure if these two threads might be helpful for ya:
https://github.com/gagle/node-properties/pull/35


not a blocker, but in case you want to get rid of any/double check some types

Reply:
Good point. I think this was existing, so I can add a type now where it was missing.

Reply:
699e5659b875f34a1310e53a02609a8ecf501c37